### ISSUE TITLE
A C++ library to parse JSON 

### ISSUE THREAD

**Issue by angelo-hub** [callstackincubator/react-native-fast-io#59]
Would you guys be open to a contribution of using [Glaze](https://github.com/stephenberry/glaze) for high performance data parsing and utilizing that within the fetch and io implementations?

Also attempting to create a package to wrap just this piece in [react-native-nitro-json](https://github.com/angelo-hub/react-native-nitro-json), to improve FPS performance on API calls parsing the data back

**Comment by grabbou** [comment]
Yes, I'd love to see that. Please feel free to start working on it!
